src/pages/typescript/installation/index.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Installation
+---
+## Installation
+
+![Installation](https://i.imgur.com/9ILjA1q.jpg)
+
+To get started yourself, the two things you will need are the TypeScript compiler and a editor that supports TypeScript.
+
+In the above screenshot, I'm installing both the compiler and [TSLint](https://github.com/palantir/tslint) (which is similar to [ESLint](https://eslint.org/)) using `npm` in [Visual Studio Code](https://code.visualstudio.com/)’s integrated terminal.
+
+### Commands
+
+> Installing TypeScript
+
+```bash
+npm i -g typescript
```

Comment:
Maybe it would be worth mentioning you can install with `--save-dev` and add a script on package.json's `scripts` section

Reply:
I will mention this as well! 👍

Reply:
Done!